### PR TITLE
feat(admin): add migrations dry run to UI sns-1867

### PR DESCRIPTION
**Pull request by dbanda**
Connects the forwards and backwards buttons of the UI to call the dry run endpoints. Removes the dummy code and actually makes dry run calls to the backend.

### CODE REVIEW

**Review by evanh — APPROVED** [review]
Was there any UI changes? If so, it would be nice to include screenshots.

File: snuba/admin/static/clickhouse_migrations/index.tsx [hunk 1]
```diff
@@ -61,6 +60,28 @@ function ClickhouseMigrations(props: { api: Client }) {
...
+      console.assert(req.dry_run, "dry_run must be set")
+      props.api
+      .runMigration(req as RunMigrationRequest)
+      .then((res) => {
+        console.log(res)
+        setSQLText(() => res.stdout);
+      })
+      .catch((err) => {
+        console.log(err)
+        setSQLText(() => JSON.stringify(err));
```
Comment (MeredithAnya) [inline snuba/admin/static/clickhouse_migrations/index.tsx:80]: we should be able to do `err.error` should we not?

Comment (dbanda) [reply snuba/admin/static/clickhouse_migrations/index.tsx:80]: This is a more generic error type and can include both client and server side response errors. For server side we can do `err.error`, client side we cannot.

**Comment by dbanda** [comment]
> Was there any UI changes? If so, it would be nice to include screenshots.

No real changes changes to the UI. Instead of dummy sql, we now show proper real dry-run SQL.

**Pull request merged**